File: atcoder/abc166B_trick_or_treat.nim
# Vicfred
# https://atcoder.jp/contests/abc166/tasks/abc166_b
# implementation
import strutils, sequtils

let
  line = stdin.readLine.split.map(parseInt)
  n = line[0]
  k = line[1]

var snukes = newSeq[int](n);

for i in 0..<k:
  let d = stdin.readLine.parseInt
  let a = stdin.readLine.split.map(parseInt)
  for j in a:
    snukes[j-1] += 1
snukes.count(0).echo

File: atcoder/abc166B_trick_or_treat.py
# uninhm
# https://atcoder.jp/contests/abc166/tasks/abc166_b
# implemtation
n, k = map(int, input().split())
 
snucks = [0]*n
 
for _ in range(k):
    input()
    for i in map(int, input().split()):
        snucks[i-1] = 1;
 
print(snucks.count(0))

